services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime.datetime,
+                         cinema_hall_id: int, movie_id: int) -> MovieSession:
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       cinema_hall_id=cinema_hall_id,
+                                       movie_id=movie_id)
+
+
+def get_movies_sessions(
+        session_date: datetime.date | None = None) -> MovieSession:
```

Comment:
The return type annotation for `get_movies_sessions` should be `QuerySet` instead of `MovieSession`, as the function returns a QuerySet of `MovieSession` objects, not a single `MovieSession` instance.